ui/admin/app/routes/_auth.tools._index.tsx

```diff
@@ -5,6 +5,7 @@ import useSWR, { preload } from "swr";
 import { ToolReferenceService } from "~/lib/service/api/toolreferenceService";
 
 import { TypographyH2 } from "~/components/Typography";
+import ErrorDialog from "~/components/knowledge/ErrorDialog";
```

Comment:
Unrelated, but can you make this a named export and update usages?

Also since this is being used in multiple packages it should probably be moved to the `~/components/composed/` directory

Reply:
Yeah can definitely do that!